nixnet/nx.py

```diff
@@ -85,6 +97,14 @@ def intf_baud_rate(self):
     def intf_baud_rate(self, value):
         _props.set_session_intf_baud_rate(self._handle, value)
 
+    @property
+    def intf_can_term(self):
+        return _props.get_session_intf_can_term(self._handle)
+
+    @intf_can_term.setter
+    def intf_can_term(self, value):
+        _props.set_session_intf_can_term(self._handle, value)
```

Comment:
I've gone ahead and merged #45.  Can you rebase this onto it and use my version of these properties?